Subject: [PATCH] fc: fix prefix path for rpc*

rpc* packages have installed files with the /usr prefix in poky, so fix 
file contexts for them.

Upstream-Status: Inappropriate [only for Poky]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/contrib/rpc.fc     |    4 ++--
 policy/modules/contrib/rpcbind.fc |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/policy/modules/contrib/rpc.fc b/policy/modules/contrib/rpc.fc
index 5c70c0c..52db849 100644
--- a/policy/modules/contrib/rpc.fc
+++ b/policy/modules/contrib/rpc.fc
@@ -9,8 +9,8 @@
 #
 # /sbin
 #
-/sbin/rpc\..*		--	gen_context(system_u:object_r:rpcd_exec_t,s0)
-/sbin/sm-notify		--	gen_context(system_u:object_r:rpcd_exec_t,s0)
+/usr/sbin/rpc\..*	--	gen_context(system_u:object_r:rpcd_exec_t,s0)
+/usr/sbin/sm-notify		--	gen_context(system_u:object_r:rpcd_exec_t,s0)
 
 #
 # /usr
diff --git a/policy/modules/contrib/rpcbind.fc b/policy/modules/contrib/rpcbind.fc
index f5c47d6..3cd9e62 100644
--- a/policy/modules/contrib/rpcbind.fc
+++ b/policy/modules/contrib/rpcbind.fc
@@ -1,6 +1,6 @@
 /etc/rc\.d/init\.d/rpcbind	--	gen_context(system_u:object_r:rpcbind_initrc_exec_t,s0)
 
-/sbin/rpcbind		--	gen_context(system_u:object_r:rpcbind_exec_t,s0)
+/usr/sbin/rpcbind	--	gen_context(system_u:object_r:rpcbind_exec_t,s0)
 
 /var/lib/rpcbind(/.*)?		gen_context(system_u:object_r:rpcbind_var_lib_t,s0)
 
-- 
1.7.5.4

